src/index.tsx

```diff
@@ -671,8 +671,26 @@ class App extends React.Component<{}, AppState> {
                     if (!e.shiftKey) {
                       clearSelection(elements);
                     }
-                    // No matter what, we select it
-                    hitElement.isSelected = true;
+                  }
+                  // No matter what, we select it
+                  hitElement.isSelected = true;
+                  // We duplicate the selected element if alt is pressed on Mouse down
+                  if (e.altKey) {
+                    const element = newElement(
```

Comment:
Can you reuse the copy paste logic instead of creating a new element from scratch by any chance?

Reply:
> Can you reuse the copy paste logic instead of creating a new element from scratch by any chance?

You mean the `JSON.stringify` workflow, or is there some helper I'm missing? I used object-spread in the [new PR](https://github.com/excalidraw/excalidraw/pull/258).